index.html

```diff
@@ -4,7 +4,7 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="/style.css" />
+		<link rel="stylesheet" href="./style.css" />
```

Comment:
Let's keep this as before keeping the path relative to the root of the domain. Adding the dot will make this less flexible. For example a page in a subfolder (about/team.html) won't be able to connect to the styles since it would be looking for a css file in the about folder instead of the root.